graph/path/shortest.go

```diff
@@ -55,13 +55,18 @@ type Shortest struct {
 	// returned the Shortest value.
 	hasNegativeCycle bool
 
-	// cycCosts holds costs between
-	// pairs of nodes to report negative
-	// cycles. cycCosts is only used
-	// if it is non-nil.
-	cycCosts map[[2]int]float64
+	// negCosts holds negative costs
```

Comment:
Why are these lines so short?

Reply:
This is for a couple of loosely related reasons; the first is that it is like this for the same reason that newsprint is in narrow column and the second is that formatting is wysiswyg in source code (that is godoc has no power here). In code, comments that explain hairy sections should make life easier; this is one way to achieve that.

Reply:
For me it makes the text harder, not easier to read. But I won't insist.

Reply:
Maybe this is a generational thing. I grew up reading newspapers.